.github/workflows/release-check.yml

```diff
@@ -11,7 +10,7 @@ jobs:
     strategy:
       matrix:
         golang:
-          - 1.18
+          - '1.20'
```

Comment:
you can remove the step `actions/cache@v3` your `actions/setup-go@v4` already caches it

Reply:
same for `.github/workflows/pr.yml`

Reply:
Awesome catch! it will make entire pipeline much cleaner.